backend/src/main/java/com/bakdata/conquery/models/forms/managed/ExternalExecution.java

```diff
@@ -92,32 +97,29 @@ public void start() {
 			User serviceUser = formBackendConfig.createServiceUser(originalUser, dataset);
 			ExternalFormBackendApi api = formBackendConfig.createApi();
 
-			final ExternalTaskState externalTaskState = api.postForm(getSubmitted(), originalUser, serviceUser, dataset);
+			super.start(executionManager);
 
-			executionManager.addState(this.getId(), new ExternalStateImpl(new CountDownLatch(0), api, serviceUser));
+			this.executionManager.addState(this.getId(), new ExternalStateImpl(ExecutionState.RUNNING, new CountDownLatch(0), api, serviceUser));
 
+			final ExternalTaskState externalTaskState = api.postForm(getSubmitted(), originalUser, serviceUser, dataset);
 			externalTaskId = externalTaskState.getId();
-
-			super.start();
 		}
 	}
 
 	private synchronized void syncExternalState(ExecutionManager executionManager) {
 		Preconditions.checkNotNull(externalTaskId, "Cannot check external task, because no Id is present");
 
-		ExternalState state = this.executionManager.getResult(this.getId());
-		final ExternalTaskState formState = state.getApi().getFormState(externalTaskId);
-
-		updateStatus(formState, executionManager);
+		Optional<ExternalState> state = this.executionManager.tryGetResult(this.getId());
```

Comment:
du verwendest hier this.executionManager obwohl du einen reingegeben bekommst. => bitte überarbeiten